rasa/train.py

```diff
@@ -135,8 +135,12 @@ def train(
     Returns:
         Path of the trained model archive.
     """
-    stories = await file_importer.get_stories()
-    nlu_data = await file_importer.get_nlu_data()
+
+    stories, nlu_data = await asyncio.gather(
```

Comment:
I think this needs `_, stories, nlu_data`

Reply:
the fingerprint should not have been in there. Good catch :+1:

Reply:
wrong catch, but good nonetheless 😉